src/inmanta/deploy/work.py

```diff
@@ -34,6 +35,17 @@
 T = TypeVar("T", bound=tasks.Task, covariant=True)
 
 
+class TaskPriority(IntEnum):
+    TERMINATED = -1
+    USER_DEPLOY = 0
+    NEW_VERSION_DEPLOY = 1
+    USER_REPAIR = 2
+    DRYRUN = 3
+    INTERVAL_DEPLOY = 4
+    FACT_REFRESH = 5
+    INTERVAL_REPAIR = 6
```

Comment:
I don't think it is much of an issue, we can change the numbers how we like because in the code we just use the task's name. @sanderr what do you think?

Reply:
Indeed, I care mostly about the relative order. I do think that a comment stating that these can be freely updated would be useful.

If we ever start writing priorities to the database it's different, but afaik we don't plan to.

Reply:
@jptrindade could you add such a comment?